protos/flyteidl/plugins/sagemaker/parameter_ranges.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package flyteidl.plugins.sagemaker;
+
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/plugins";
```

Comment:
Does this still correctly generate a `"github.com/lyft/flyteidl/gen/pb-go/flyteidl/plugins/sagemaker"` package following the file path? or will these end up in `"github.com/lyft/flyteidl/gen/pb-go/flyteidl/plugins"` package ?

Reply:
The package is generated at the right location `"github.com/lyft/flyteidl/gen/pb-go/flyteidl/plugins/sagemaker"`